aidial_adapter_bedrock/llm/message.py

```diff
@@ -66,15 +66,10 @@ def _parse_assistant_message(
     if content is None and function_call is not None and tool_calls is None:
         return AIFunctionCallMessage(call=function_call)
 
-    if content is None and function_call is None and tool_calls is not None:
+    if function_call is None and tool_calls is not None:
```

Comment:
why did this check go away?

Reply:
Because Claude can return content in this type of messages

It looks something like that:

```python
{
    "role": "assistant",
    "content": "Okay, let's get the weather for San Francisco:",
    "tool_calls": [
        {
            "index": 0,
            "id": "toolu_bdrk_01KMwM3ZXeyYQJTnmDSaBNqb",
            "type": "function",
            "function": {
                "name": "get_weather",
                "arguments": '{"location": "San Francisco"}',
            },
        }
    ],
},
```

Reply:
Then we should put the content inside `AIToolCallMessage`, otherwise the information is lost.
We want Claude to see this content too in the input messages, right?

Reply:
Fixed, added content to this messages